work/src/org/kuali/kfs/module/ar/businessobject/ContractsGrantsInvoiceDetail.java

```diff
@@ -159,29 +159,29 @@ public void setCumulative(KualiDecimal cumulative) {
      *
      * @return Returns the balance.
      */
-    public KualiDecimal getBalance() {
+    public KualiDecimal getBudgetRemaining() {
         // Balance = Budget-Cumulative
```

Comment:
comment is a bit out of date; I'd probably remove altogether and let the code speak for itself